extensions/amp-accordion/1.0/README.md

```diff
@@ -25,7 +25,17 @@ import {defineElement as defineBentoAccordion} from '@bentoproject/accordion';
 defineBentoAccordion();
 ```
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script type="nomodule" src="https://cdn.ampproject.org/bento.js"></script>
+<script type="module" src="https://cdn.ampproject.org/v0/bento-accordion-1.0.mjs"></script>
+<script type="nomodule"  src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-accordion-1.0.css"/>
```

Comment:
Any reason for using self-closing tag for `link` elements?

```suggestion
<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-accordion-1.0.css">
```

It's only relevant for XHTML which is largely from a bygone era.

Reply:
No idea where that came from...removed.